src/runtime_src/tools/xclbinutil/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 # Signing xclbin images are currently only support on Linux
 
+# cmake 3.19 and after emits warning about this policy not being set
+if((CMAKE_VERSION VERSION_GREATER 3.19) OR (CMAKE_VERSION VERSION_EQUAL 3.19))
```

Comment:
https://cmake.org/cmake/help/latest/command/if.html#version-greater-equal

Reply:
VERSION_GREATER_EQUAL macro support is added from 3.7 version of cmake and on OSes like Ubuntu 16.04 its not available resulting in cmake build failure. As this is 2021.1 branch Ubuntu 16.04 build needs to be supported

Reply:
I am always surprised  by our JurassiX computing. Does XRT as it just even compile on Ubuntu 16.04 since nowadays there is a lot of C++17 in the code? But perhaps not in 2021.1? @stsoe 
So, does this means that in the `master` branch of XRT `VERSION_GREATER_EQUAL` is used instead?

Reply:
Yes in master branch we can use VERSION_GREATER_EQUAL macro as Ubuntu 16.04 support is deprecated in later branches. Also pipeline tests suggests that 2021.1 branch compiles successfully on 16.04 Ubuntu.